community/cypher/cypher/src/test/scala/org/neo4j/cypher/RootPlanAcceptanceTest.scala

```diff
@@ -146,7 +146,8 @@ class RootPlanAcceptanceTest extends ExecutionEngineFunSuite {
   }
 
   test("EstimatedRows should be properly formatted") {
-    given("match n return n").planDescripton.getArguments.get("EstimatedRows") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedOperatorCardinality") should equal(0)
+    given("match n return n").planDescripton.getArguments.get("EstimatedProducedRows") should equal(0)
```

Comment:
More generally I think it is good to include this information it may be helpful for us and also explain slow queries better to power users.